wandb/cli.py

```diff
@@ -775,7 +775,7 @@ def run(ctx, program, args, id, resume, dir, configs, message, name, notes, show
 @cli.command(context_settings=RUN_CONTEXT, name="docker-run")
 @click.pass_context
 @click.argument('docker_run_args', nargs=-1)
-@click.option('--help')
+@click.option('--help', is_flag=True, defualt=False, help='Show docker run helper.')
```

Comment:
There appears to be a typo with "defualt" here.

Thanks for finding this.
I think a better fix is to add help="Wrap docker run with W&B api key" to the `@cli.command()` line and remove the `help` flag altogether.

Reply:
`help` flag is still used in the code for printing docker help text, so lets keep it for now. I addressed the other two comments.

Reply:
`defualt` => `default`

Reply:
yes its addressed in `25e2466`